models.py

```diff
@@ -151,6 +161,10 @@ def forward(self,
       encoder_states: Encoder outputs with shape
         (input seq len, batch size, hidden size).
       attention_mask: Attention mask (batch size, input seq len).
+      indices: indices in the extended vocab.
```

Comment:
Indices of what? What are they used for. Indices is a very general word -- and it would be good to document it in more detail for future use